### PR TITLE
+angular.dev

### PR DESCRIPTION
**Pull request by and-ri**
https://ossinsight.io/analyze/angular/angular

### CODE REVIEW

File: projects/angular.dev/package.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+versions:
+  github: angular/angular
+dependencies:
+  nodejs.org: ^20
+build:
+  dependencies:
+    npmjs.com: ^10
+    curl.se: '*'
+  script:
+    - curl -L https://registry.npmjs.org/@angular/cli/-/cli-{{version}}.tgz --output cli-{{version}}.tgz
```
Comment (jhheider) [inline projects/angular.dev/package.yml]: why can't this url be used as distributable? you should be able to use `.` as the path once its upacked, i believe.

Comment (and-ri) [reply projects/angular.dev/package.yml]: It looks like #3561 
There I was getting an error when using `.`

Comment (and-ri) [reply projects/angular.dev/package.yml]: But I'll try now

**Comment by and-ri** [comment]
If we do not use the `.tar.gz.` archive, we get a symbolic link to the build directory in `{{prefix}}/libexec/lib/node_modules/@angular`

**Comment by jhheider** [comment]
try the confusingly-named `--install-links` (which they've inverted the meaning of before :( )

**Pull request merged**